manifests/vanilla/vsphere-csi-driver.yaml

```diff
@@ -236,7 +236,7 @@ spec:
       dnsPolicy: "Default"
       containers:
         - name: csi-attacher
-          image: k8s.gcr.io/sig-storage/csi-attacher:v3.5.0
+          image: k8s.gcr.io/sig-storage/csi-attacher:v4.0.0
```

Comment:
[v4.0.0](https://github.com/kubernetes-csi/external-attacher/releases/tag/v4.0.0) has CSIMigration feature turned on by default. Will that affect anything for the CSI driver?

Reply:
There is no impact on the driver as such.